Feature/doc-691: Document new Comments success and error callbacks (#1731)
tinymce/tinymce-docs: advanced/configuring-comments-callbacks.md

```js
{
  conversationUid: string, // the new conversation uid

>   // optional error callback which will be executed in case the new conversation could not be created

Reviewer: ```suggestion
  // Optional error callback which will be run if the new conversation could not be created
```

  onError: function (err) { ... },

>   // optional success callback which will be executed when the new conversation is successfully created

Reviewer: ```suggestion
  // Optional success callback which will be run when the new conversation is successfully created
```

Reviewer 2: How come we say "if" for failures and "when" for successes, is that deliberate?

Reviewer: @ashfordneil not intentional. Now I'm overthinking this issue.
